api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -208,7 +210,20 @@ private Workspace createWorkspaceImpl(Workspace workspace) {
     DbUser user = userProvider.get();
 
     // Note: please keep any initialization logic here in sync with cloneWorkspaceImpl().
-    final String billingProject = claimBillingProject(user, accessTier);
+    String billingProject;
+    if(workbenchConfigProvider.get().featureFlags.enableFireCloudV2Billing) {
+      // If v2 Billing is enabled, we will call FireCloud directly to create one.
+      billingProject = createBillingProjectName(workbenchConfigProvider);
```

Comment:
nit: I like having the class name prefixed for static methods so its clear right away that this functionality is from a different service/class.